src/Common/src/CoreLib/Interop/Unix/System.Native/Interop.GetTimestamp.cs

```diff
@@ -8,10 +8,10 @@ internal static partial class Interop
 {
     internal static partial class Sys
     {
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetTimestampResolution")]
-        internal static extern bool GetTimestampResolution(out long resolution);
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetTimestampResolution", BestFitMapping = false, ExactSpelling = true)]
```

Comment:
`BestFitMapping = false` is unnecessary

Reply:
Sorry, thought I had gotten these, but they were missed in the commit.

Should be fixed for all 4 P/Invoke signatures now.